doc/source/whatsnew/v0.16.1.txt

```diff
@@ -80,3 +80,5 @@ Bug Fixes
 
 - Bug in ``Series.quantile`` on empty Series of type ``Datetime`` or ``Timedelta`` (:issue:`9675`)
 - Bug in ``where`` causing incorrect results when upcasting was required (:issue:`9731`)
+
+- Bug in ``ParserBase.convert_to_nd_arrays`` when called by ``DataFrame.read_fwf`` (:issue:`9266``) 
```

Comment:
needs to be moved to 0.17.0. Say something like ``fixed bug in csv parsing when using a converting that specified uint8``